### DISCUSSION TITLE
Having Error with cQube installation

### DISCUSSION THREAD

**Discussion by priya23595** [Sunbird-cQube/community#788] [Q&A]
Hi
I tried to install cQube on my local machine. I am facing with issues.
Following are the steps that i took

Install ansible
docker is already
 installed
docker-compose was already present
cloned cqube-devops and ran install script
Following is the screenshot for the error
![Screenshot from 2023-06-20 15-05-44](https://github.com/Sunbird-cQube/community/assets/136596601/b414cee9-44f3-46db-98f1-831b463b72fe)

**Comment by RNarmatha** [comment]
Hi @priya23595 - Kindly checkout the latest branch
